build/patches/add-support-for-ISupportHelpAndFeedback.patch

```diff
@@ -0,0 +1,44 @@
+From: uazo <uazo@users.noreply.github.com>
+Date: Mon, 17 May 2021 12:30:12 +0000
+Subject: add-support-for-ISupportHelpAndFeedback.patch
```

Comment:
```suggestion
Subject: add-support-for-ISupportHelpAndFeedback
```

Reply:
fixed in https://github.com/bromite/bromite/pull/857/commits/e65cfb40ec64cba44d61323d3a35a0b02178bd95